Source/WebCore/dom/ScriptElement.h

```diff
@@ -157,6 +157,7 @@ class ScriptElement {
 
     // https://w3c.github.io/trusted-types/dist/spec/#slots-with-trusted-values
     String m_trustedScriptText { emptyString() };
+    bool m_childrenChangedByApi : 1 { false };
```

Comment:
API should be kept all capitals, but not sure that "API" is what we want to say here, maybe "Script" is better. I guess we do use API for `ContainerNode::ChildChange::Source::API`, so we can stick with that if you like.

Writing `: 1` here is not a good optimization. This bit won’t be packed with other bits unless they are consecutive. So we should either move this up above with the other single bit field booleans, or we should omit the `: 1`.